QSB/EventsCore/QSBEvent.cs

```diff
@@ -1,8 +1,9 @@
 ﻿using QSB.Messaging;
+using QSB.Player;
 using QSB.TransformSync;
 using UnityEngine.Networking;
 
-namespace QSB.Events
+namespace QSB.EventsCore
```

Comment:
not sure what the "Core" stuff is, but how about calling the namespace `QSB.Events.Core`? That's more "conventional"